server/server.go

```diff
@@ -141,7 +142,7 @@ func NewEmulatorServer(logger *logrus.Logger, conf *Config) *EmulatorServer {
 		"FlowServiceAccount": chain.ServiceAddress().HexWithPrefix(),
 		"FlowToken":          fvm.FlowTokenAddress(chain).HexWithPrefix(),
 		"FungibleToken":      fvm.FungibleTokenAddress(chain).HexWithPrefix(),
-		"FlowFees":           fvm.FlowFeesAddress(chain).HexWithPrefix(),
+		"FlowFees":           environment.FlowFeesAddress(chain).HexWithPrefix(),
```

Comment:
is there a logic for what is on fvm and what is on environment?

Reply:
I don't know why it was moved and just adjusted the code here to compile, but will ask the team.

Reply:
The FlowFeesAddress was moved together with its only usage in fvm/environment during a refactor.

There was [preparation done](https://github.com/onflow/flow-go/pull/3218) to remove all of these actually.  There will be a different way to get them soon: `systemcontracts.SystemContractsForChain(chain.ChainID())` which should clean this up.